app/models/ont/run.rb

```diff
@@ -7,6 +7,8 @@ class Run < ApplicationRecord
 
     has_many :flowcells, foreign_key: :ont_run_id, inverse_of: :run, dependent: :destroy
 
+    validates :instrument_name, presence: true
```

Comment:
Sorry. I should have spotted this earlier but again instrument name will be hardcoded.